### PR TITLE
Added option to hardcode load balancer IP for allocator.

### LINKED ISSUE #1709: Helm chart: allow to specify loadBalancerIP
**Is your feature request related to a problem? Please describe.**
Helm chart does not allow to specify loadBalancerIP for Agones services, which causes problems when you recreate them, potentially changing the IP, which may require updating DNS/certs/configs.

**Describe the solution you'd like**
Allow providing `loadBalancerIP` that would be used if the `serviceType` is `LoadBalancer`.

### PR DESCRIPTION
**Pull request by devloop0**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind breaking
> /kind bug
> /kind cleanup
> /kind documentation

/kind feature

> /kind hotfix

**What this PR does / Why we need it**:
Adds an agones.allocator.http.loadBalancerIP Helm configuration parameter to hardcode load balancer IP (if provided).

**Which issue(s) this PR fixes**:

Closes #1709

**Special notes for your reviewer**:

### CODE REVIEW

File: install/helm/agones/values.yaml [hunk 1]
```diff
@@ -124,6 +124,7 @@ agones:
     http:
       port: 443
       serviceType: LoadBalancer
+      loadBalancerIP: ""
```
Comment (pooneh-m) [inline install/helm/agones/values.yaml:127]: Can we add that in the [helm installation](https://github.com/googleforgames/agones/blob/master/site/content/en/docs/Installation/Install%20Agones/helm.md) documentation as well? 

Also, we should note that it only works if the k8s provider supports that.

Comment (devloop0) [reply install/helm/agones/values.yaml:127]: Done!

**Comment by devloop0** [comment]
I manually tested this change as follows:
1. Created a new cluster following the directions here: https://agones.dev/site/docs/installation/creating-cluster/gke/

2. Reserved a static, regional IP <LB-IP> for a load balancer in the same region as my cluster

3. `helm upgrade --install --namespace=agones-system --set agones.image.tag=1.9.0-1fc5fc2 --set agones.allocator.http.loadBalancerIP="<LB-IP>" agones ./install/helm/agones/`
(The image tag corresponds to the previous successful build for this PR)

4. Waited to make sure that the external IP of the agones-allocator load balancer matched <LB-IP>

5. Generated a client cert that the allocator would recognize from here: https://agones.dev/site/docs/advanced/allocator-service/#client-certificate

6. *Just extracted the existing CA crt* (Reversing the last command from here: https://agones.dev/site/docs/advanced/allocator-service/#server-tls-certificate)
`kubectl get secret allocator-tls-ca -o json -n agones-system | jq '.data["tls-ca.crt"]' -r | base64 -d > ca.crt`

7. Confirmed that allocating using the client gave no cert errors (as detailed here: https://agones.dev/site/docs/advanced/allocator-service/#send-allocation-request)

**Comment by pooneh-m** [comment]
Looks great! Thank you.

**Pull request merged**